roles/app/templates/post-receive.j2

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-APP_PATH="$HOME/apps/openfoodnetwork"
+APP_PATH="$HOME/apps/ofn_america"
```

Comment:
No. It certainly shouldn't be changed universally. Didn't mean to push this change, and have since realized that it makes sense to just leave the app name as universal. At least until there's a reason not to.